ClipperLibBaseDockerfile

```diff
@@ -1,40 +1,17 @@
-FROM frolvlad/alpine-gxx
+FROM ubuntu:16.04
```

Comment:
From an ease of development standpoint, this change of base image is a plus. Any concerns about the difference in image size and overhead?

Reply:
Not at this point. We can always build a minimal size docker image on alpine in the future. But that doesn't seem super important right now.

Reply:
Imo there shouldn't really be any since this is a base file used for frontend containers that we won't be replicating as frequently as model containers. Merging...